test/clt-tests/test-enable-disable-buddy-plugin/test-enable-disable-buddy-plugin.rec

```diff
@@ -0,0 +1,48 @@
+––– block: ../base/start-searchd –––
+––– input –––
+mysql -h0 -P9306 -e "SHOW QUERIES"
+––– output –––
++------+--------------+----------+-----------------+
+| id   | query        | protocol | host            |
```

Comment:
Merge your branch with master. Yesterday here we add `time` column

Reply:
Done